contracts/contracts/vault/OETHPlumeVaultCore.sol

```diff
@@ -0,0 +1,51 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import { StableMath } from "../utils/StableMath.sol";
+import { OETHVaultCore } from "./OETHVaultCore.sol";
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IStrategy } from "../interfaces/IStrategy.sol";
+
+/**
+ * @title OETH Plume VaultCore Contract
+ * @author Origin Protocol Inc
+ */
+contract OETHPlumeVaultCore is OETHVaultCore {
+    using SafeERC20 for IERC20;
```

Comment:
nit we don't need these libraries here. They will be included in the inherited contracts when needed.